src/Disks/ObjectStorages/AzureBlobStorage/AzureObjectStorage.cpp

```diff
@@ -207,7 +209,7 @@ void AzureObjectStorage::applyNewSettings(const Poco::Util::AbstractConfiguratio
 {
     auto new_settings = getAzureBlobStorageSettings(config, config_prefix, context);
     settings.set(std::move(new_settings));
-
+    applyRemoteThrottlingSettings(context);
```

Comment:
There is also another way, but not sure if it is any better, probably not better at all: may be rename it to `applyThrottlingSettings` and use `isRemote()` method inside to check to decide which throttler from context we need to get. As for not there is only remote option, then `else` part of `isRemote()` will be noop.

Reply:
I decided to just rename it in settings, so this also is consistent now without even checking `isRemote()`